manifests/install.pp

```diff
@@ -195,4 +195,9 @@
       fail("Installation method ${grafana::install_method} not supported")
     }
   }
+
+  if !empty($grafana::ldap_servers) {
```

Comment:
I'm not sure if this whole if block is a good idea. the ruby-toml package isn't really grafana specific, so people might already declare it in another module. What about a new parameter `Boolean $manage_toml` that wraps `ensure_package()` from stdlib?

Reply:
Sure, no problem. Have had discovered in the morning that I need ruby-toml, as the generating part in the custom-type moves then from puppetserver to agents.

But I was also thinking if it is actually worth the effort to add ruby-toml as dependency on agent-side - or using an ordinary puppet-template instead to generate the config. Not sure.